app/src/views/search-engine/index.js

```diff
@@ -14,6 +15,8 @@ const SearchEngine = () => {
   const [search, setSearch] = useState('')
   const debouncedSearch = useDebounce(search, 1000)
   const getArticlesQuery = useQuery(GET_ARTICLES, { variables: { search: debouncedSearch } })
+  const classes = styles()
+  const DOC_URL = 'https://www.guadalupe.gob.mx/nuevo-reglamento-de-transito-homologado/'
```

Comment:
This is okay as a short term fix, however in the long term you and @jaimehisao should figure out how this value could be determined dynamically. Once you add more jurisdictions, it does not make sense to hardcode this URL.

Reply:
Totally agree. Once having a DB where we store the documents, fetching its information is what we should be doing.
Something like:

```
const document = await fetch('/document/id');
document.URL
document.name
...
```

That way we'd achieve a more dynamic frontend. But yeah, that's something to look into.